vocabularies/UI.json

```diff
@@ -1609,7 +1617,8 @@
             "@Core.Description": "Elementary building block that represents a piece of data and/or allows triggering an action",
             "@Validation.ApplicableTerms": [
                 "UI.Hidden",
-                "UI.Importance"
+                "UI.Importance",
+                "HTML5.CssDefaults"
```

Comment:
If we remove the "e.g. in UI.DataFieldAbstract" in HTML5 Vocab. We may add a comment to the long description here

Reply:
I was not sure where to put here a long description - maybe to all the applicable terms?

Reply:
I meant in the LongDescription of the DataFieldAbstract above (not visible in the comment)

Reply:
OK - I made a proposal